docs/installation.rst

```diff
@@ -231,8 +231,9 @@ Cloud installation
 
 Installation on AWS for distributed computing
 ---------------------------------------------
-We use CGCloud_ to provision instances and clusters in AWS. Thorough documentation of CGCloud_ can be found
-in the CGCloud-core_ and CGCloud-toil_ documentation. Brief steps will be provided to those
+We use CGCloud_ and the Toil AWS provisioner to provision instances and clusters in AWS. Thorough documentation
```

Comment:
Give more importance to the AWS provisioner. CGCloud is on the way out, mention it last.